app-emulation/ganeti/ganeti-2.15.2-r10.ebuild

```diff
@@ -204,6 +204,8 @@ PATCHES+=(
 	"${FILESDIR}/${PN}-2.15-dont-invert-return-values-for-man-warnings.patch"
 	"${FILESDIR}/${PN}-2.15-respect-HFLAGS.patch"
 	"${FILESDIR}/ganeti-2.15.2-bdev_py.patch"
+	"${FILESDIR}/ganeti-2.15.2-check-man-warnings.patch"
```

Comment:
${PN} in these.

Reply:
I prefer non-${PN} these days, i would rather convert all the older patches to use the package name directly. Makes grepping for patch usage easier.